generators/server/templates/dotnetcore/src/Project/Infrastructure/DatabaseStartup.cs.ejs

```diff
@@ -16,7 +16,18 @@ using System;
 using <%= namespace %>.Data;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.Extensions.Hosting;
+<%_ switch(databaseType) {
```

Comment:
What if the user select Oracle? I don't see the according imports

Reply:
Is not needed, Oracle.EntityFrameworkCore adds few methods ( like UseOracle ) to Microsoft.EntityFrameworkCore namespace which is allready imported